djiffy/migrations/0001_initial.py

```diff
@@ -1,55 +1,64 @@
-# Generated by Django 3.2.20 on 2024-08-12 19:33
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-17 18:18
+from __future__ import unicode_literals
 
 from django.db import migrations, models
 import django.db.models.deletion
+import jsonfield.fields
```

Comment:
This is coming back to me vaguely, I bet I wanted to squash so I could get rid of the extra dependency.